js/ui/date_box/ui.date_box.strategy.js

```diff
@@ -90,12 +90,6 @@ const DateBoxStrategy = Class.inherit({
         this._widget && this._widget.option('value', this.dateBoxValue());
     },
 
-    _valueChangedHandler: function(args) {
```

Comment:
Unused code. This handler is used only in a strategy with a calendar, but there the handler is redefined.